Engines/Wine/Verbs/mfWorkaround/script.js

```diff
@@ -0,0 +1,113 @@
+const Wine = include("engines.wine.engine.object");
+const Resource = include("utils.functions.net.resource");
+const { Extractor } = include("utils.functions.filesystem.extract");
+const { ls, cp } = include("utils.functions.filesystem.files");
+//const { getGithubReleases } = include("utils.functions.net.githubreleases"); commented out because it's useless until z0z0z will use releases
+
+const Optional = Java.type("java.util.Optional");
+const Regedit = include("engines.wine.plugins.regedit");
+const Regsvr32 = include("engines.wine.plugins.regsvr32");
+const OverrideDLL = include("engines.wine.plugins.override_dll");
+
+/**
+ * Verb to install Media Foundation workaround
+ * see: https://github.com/z0z0z/mf-install
+ */
+class mfWorkaround {
+    constructor(wine) {
+        this.wine = wine;
+    }
+
+    /**
+     * Sets the used mfWorkaround version
+     *
+     * @param {string} mfWorkaroundVersion The version of mfWorkaround to downlaod
+     * @returns {mfWorkaround} The mfWorkaround object
+     */
+    withVersion(mfWorkaroundVersion) {
+        this.mfWorkaroundVersion = mfWorkaroundVersion;
+
+        return this;
+    }
```

Comment:
If `mfWorkaroundVersion` can't be used I would recommend removing the field and the method for now because it is more confusing that useful